configuration/pom.xml

```diff
@@ -18,6 +18,14 @@
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+        </dependency>
```

Comment:
from doc `Returns an unmodifiable view of the specified map.` this give us only **view**